src/hooks/init/check-update.ts

```diff
@@ -130,15 +133,25 @@ const hook: Hook.Init = async function ({config}) {
   const debug = makeDebug('update-check')
   const versionFile = join(config.cacheDir, 'version')
   const lastWarningFile = join(config.cacheDir, 'last-warning')
+  const scope = config.name.split('/')[0];
 
   // Destructure package.json configuration with defaults
   const {
-    authorization = '',
     message = '<%= config.name %> update available from <%= chalk.greenBright(config.version) %> to <%= chalk.greenBright(latest) %>.',
-    registry = config.npmRegistry ?? 'https://registry.npmjs.org',
-    timeoutInDays = 60,
+    registry = config.npmRegistry ?? getRegistryUrl(scope), // Use custom registry or fallback to 1) registry set for the scope, 2) default registry in the npmrc, or 3) the default registry
+    timeoutInDays = 7,
```

Comment:
why change the default `timeoutInDays`?

Reply:
Fair question as it's not really relevant to this change in general, but this felt like a much more reasonable default for most tools. It's configurable so not a big deal but two months is a long time to wait (by default) to notify users of a new version.